### DISCUSSION TITLE
Beatmap never finishes if user logs out during gameplay

### DISCUSSION THREAD

**Discussion by Susko3** [ppy/osu#16753] [Q&A] [answered]
https://user-images.githubusercontent.com/16479013/152243363-f39fbcde-3661-4eaa-b9c3-ed4dba3e1c46.mp4

Self-explanatory, looks like the score panel never loads. (Signing in at this point will not fix it.)

If a different user logs in, the score panel loads normally (with the original user displayed as the player), and a "Score failed to submit" notification will appear. (I was using dev server for testing this one 😉)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16757
